tests/kafkatest/services/verifiable_producer.py

```diff
@@ -55,16 +79,32 @@ def _worker(self, idx, node):
                     elif data["name"] == "producer_send_success":
                         self.acked_values.append(int(data["value"]))
 
-    @property
-    def start_cmd(self):
-        cmd = "/opt/kafka/bin/kafka-verifiable-producer.sh" \
+    def _kafka_dir(self, node):
+        if node.version == KafkaVersion.TRUNK:
+            return "/opt/kafka"
+        else:
+            return "/opt/kafka-" + node.version
+
+    def start_cmd(self, node):
+
+        cmd = ""
+        if node.version == KafkaVersion.V_0_8_2_1:
+            cmd += "for file in /opt/kafka/tools/build/libs/kafka-tools*.jar; do CLASSPATH=$CLASSPATH:$file; done; "
```

Comment:
Hmm, why do we have all of this? And why would a 0.8.2.1 version reference all this? Is this because we didn't have the tool in that version of Kafka?

This snippet concerns me for two reasons. First, having to do this custom stuff for different versions seems like it'll end up being a mess. It is at least isolated to services, but I'd like to avoid special-case code if possible. Second, if you mix in jars from trunk w/ previous versions, you're probably going to get mismatches. Is this intended to work because we still use 0.8.2.1 jars *except* for the additions from kafka-tools*.jar?